### PR TITLE
support tvOS

### PR DESCRIPTION
**Pull request by KenShih522**
## Checklist

- [x] New Extension
- [ ] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by KenShih522** [comment]
support tvOS

**Comment by Khalian** [comment]
I am confused as to what this extension does. You want to disable or enable tvOS support for addToPasteboard. If so why?

**Comment by KenShih522** [comment]
Hi, i am developing an application for tvOS. I found that I can not compile your source code on tvOS.
It seems like UIPasteboard can not be used on tvOS because they marked it __TVOS_PROHIBITED.

File: Sources/StringExtensions.swift [hunk 1]
```diff
@@ -450,11 +450,15 @@ extension String {
         return false
     }
     
+    #if os(iOS)
```
Comment (Khalian) [inline Sources/StringExtensions.swift]: Lets add a comment on top of this as to why this change is being made ("UIPasteboard is __TVOS_PROHIBITED and hence only supported in iOS)"

**Comment by Khalian** [comment]
Also would you mind changing the description of the commit message to something along the lines of:-

"Defining addToPasteboard StringExtension to only be usable in IOS

Reason : UIPasteboard is __TVOS_PROHIBITED"

**Comment by Khalian** [comment]
Before we merge, @lfarah I think swift lint is acting up again. Do you have any context on this ?

**Comment by lfarah** [comment]
@Khalian newest Xcode doesn't support Swift 2.3

**Comment by lfarah** [comment]
@KenShih522 do you intend to use Swift 2.3 or that was a mistake?

**Comment by KenShih522** [comment]
Yes, I am currently using swift 2.3 to develop my project.

**Comment by Khalian** [comment]
@goktugyil should we merge this ?

**Comment by goktugyil** [comment]
tvOS support constantly fails while we add new things. IS there like an auto test that checks for that too?

**Comment by Khalian** [comment]
@goktugyil We can raise autotests for tvos as a separate issue. Resolving this for now.
